src/test/java/com/google/cloud/tools/maven/run/RunAsyncMojoTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2016 Google LLC. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.maven.run;
+
+import junitparams.JUnitParamsRunner;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitParamsRunner.class)
+public class RunAsyncMojoTest {
+  //
```

Comment:
Can delete this file, unless you intend on uncommenting this (if you keep the file, change the copyright year to 2018)